### DISCUSSION TITLE
CMS content management

### DISCUSSION THREAD

**Discussion by davegariepy** [parcel-bundler/parcel#4693] [General] [answered]
Where do you store content?

Is it possible to use a headless CMS?

If using third party API to get data, is that pulled in when building and deployed statically or is the api request happening in the browser when the user loads the page. 

I am comparing to Gatsby where the way I understand is that you can add a source and when the project is built it pulls in all the data and creates static pages through GraphQL

I don’t see anything like this mentioned in parcel so are people just storing the content for blogs or whatever right in the code repo?

Thanks

**Comment by mischnic** [comment, marked as answer]
Parcel is not a CMS (like 11ty or Gatstby), but a (Javascript) Bundler like Webpack, Browserify or Rollup.
